website/content/en/docs/building-operators/golang/migration.md

```diff
@@ -68,7 +68,7 @@ operator-sdk init --domain example.com --repo github.com/example-inc/memcached-o
 
 ## Check if your project is multi-group
 
-Before we start to create the APIs, check if your project has more than one group such as : `foo.example.com/v1` and `crew.example.com/v1`. If you intend to still working on with multiple groups in your project, then add the line 
```

Comment:
Sidenote, maybe we ought to add these style checks in our linter to make reviews easier.

Reply:
Let's track this need.